gems/smithy/spec/fixtures/shapes/model.smithy

```diff
@@ -0,0 +1,115 @@
+$version: "2"
+
+namespace smithy.ruby.tests
+
+@paginated(inputToken: "nextToken", outputToken: "nextToken", pageSize: "pageSize")
+service ShapeService {
+    version: "2018-10-31"
+    resources: [
+        SomeResource
+    ]
+    operations: [
+        SomeOperation
+    ]
+}
+
+operation SomeOperation {
+    input: SomeOperationInput
+    output: SomeOperationOutput
+    errors: [SomeError]
+}
+
+@input
+@documentation("This is a documentation")
+structure SomeOperationInput {
+    bigDecimal: BigDecimal
+    bigInteger: BigInteger
+    blob: Blob
+    boolean: Boolean
+    byte: Byte
+    double: Double
+    enum: SomeEnum
+    float: Float
+    integer: Integer
+    intEnum: SomeIntEnum
+    long: Long
+    short: Short
+
+    @required
+    string: String
+
+    timestamp: Timestamp
+}
+
+@output
+structure SomeOperationOutput {
+    list: SomeList
```

Comment:
Naming nit, can probably drop the Some prefixing. It's fine for operation or structure name.